docs/DeveloperGuide.adoc

```diff
@@ -294,36 +295,55 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
-|Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+| Priority | As a... | I want to... | So that I can...
+| `* * *` | familiar with command line | type commands into a command line interface | perform tasks faster
+| `* * *` | university student | use flashcards to remember things easily | prepare better for my exams
+| `* * *` | meticulous student | set priorities of each subject I am studying  | study subjects in an orderly manner
+| `* * *` | forgetful student | set exam reminders   | remember when to start studying for my exams
+| `* * *` | busy student  | exit the flashcard mode at any time  | perform other tasks
+| `* * *` | meticulous student  | search for a particular flashcard | review a specific flashcard whenever I want
+| `* * *` | new user | key in part of a command and let the program suggest the rest | key in commands more easily
+| `* * *` | student | see a timer next to each flashcard  | know how long I&#39;ve spent answering the current question
+| `* * *` | student studying multiple subjects | categorize my flashcards under different modules | study them in such manner
+| `* * *` | meticulous student | sort my subjects manner.  | study subjects in an orderly
+| `* * *` | meticulous student | make use of colour codes  | highlight and know my weaknesses
+| `* * *` | advanced user | search through my content   | find subjects/topics easily
+| `* * *` | meticulous student,   | use the calendar to plan my study |  so that I will not miss any subject
+| `* * *` | meticulous student | view what tasks are overdue | know what I have not done
+| `* * *` | meticulous student | a mock test timer to simulate exam situation | be prepared for my upcoming exams
+| `* * *` | meticulous student | label difficult questions | the flash cards will appear more frequently for better knowledge absorption
+| `* * *` | student | move backward through the flashcards  | recall the previous question/answer
+| `* *`  | meticulous student | view the overall statistic | monitor my performance overtime
+| `* *`  | meticulous student | use the scoring system  | know how well prepared I am
+| `* *`  | As a user | hide the answer until I finish all the questions | remember better
+| `* *`  | meticulous student | check the correctness of my answer | know whether my answer was right
+| `* *`  | busy student | use the night mode  | study in a dark environment
```

Comment:
"Night owl" or "student that studies at night" might be a better user type/role than "busy student" here.